web/js/bundle.js

```diff
@@ -7337,6 +7370,11 @@ function planetColor(d) {
 	return colorToRgb(c)
 }
 
+function wreckColor(d) {
+	let c = d.data.players[d.player].color
+	return colorToRgb([ c[0] * 0.2, c[1] * 0.2, c[2] * 0.2 ])
```

Comment:
why the 0.2 constant?

Reply:
it reduces brightness and saturation of the color to make it less disturbing, less significant in the visualization